cgi-bin/DW/External/Site.pm

```diff
@@ -96,7 +130,8 @@ sub get_site {
                 map { lc $_ }
                 split( /\./, $site );
 
-    return $domaintosite{"$parts[-2].$parts[-1]"} || DW::External::Site::Unknown->accepts( \@parts ) || undef;
+    return $domaintosite{"$parts[-2].$parts[-1]"} || $domaintosite{"$parts[-1]"} || 
```

Comment:
If I'm reading this right, I think we only need the list of aliases (above), and the accepts (below). 

The main thing being that once the $domaintosite{"parts[-1]") matches, we return the relevant DW::External::Site instance directly -- never end up calling DW::External::Site::Unknown->accepts 

(huh in fact it seems like we never actually do call any of the DW::External::Site->accepts() no matter what).